packages/ui/src/components/Chip.svelte

```diff
@@ -14,9 +14,13 @@
 -->
 <script lang="ts">
   import { createEventDispatcher } from 'svelte'
-  import { IconClose, ButtonIcon } from '..'
+  import { IconClose, ButtonIcon, LabelAndProps, tooltip } from '..'
 
   export let label: string
+  export let size: 'small' | 'min' = 'small'
+  export let isRemovable: boolean = false
+  export let backgroundColor: string | undefined = undefined
+  export let showTooltip: LabelAndProps | undefined = undefined
```

Comment:
I think it would be better to name it something like just `tooltip`. Buy we already named it `showTooltip` for `Button`, `StyleButton` and 24 other components.

Reply:
Let's rename to `tooltip`

Reply:
Maybe it would be better to make another PR where we rename all tooltip properties in all components for consistency?

I can do it very quickly.